share/postgresql-setup/postgresql_pkg_tests.sh.in

```diff
@@ -4,11 +4,23 @@
 
 # This file is to be sourced.
 
+try_random_port ()
+{
+    _port=$RANDOM
+    if test -n "$_port" && eval '_port=$(( $_port + 32000 ))' 2>/dev/null; then
+        eval 'echo "$_port"'
```

Comment:
I am wondering - why `eval echo` instead of just `echo`?

Reply:
Good question :-) I tried to silence ShellCheck warning (used but unset _port variable).  But I can avoid using that...